src/OpenLoco/src/Ui/Windows/Options.cpp

```diff
@@ -1226,55 +1226,18 @@ namespace OpenLoco::Ui::Windows::Options
 
 #pragma mark - Widget 11
 
-        static std::vector<int> get_available_tracks()
+        /**
+         * Returns a vector of music track IDs that this dropdown is to be populated with.
+         */
+        static auto getTracks()
```

Comment:
I think you can just call `Audio::makeSelectedPlaylist()` in the 2 places this variable is used; adding another level of redirection doesn't achieve anything here (and also we don't use multiline comments afaik, and *especially* when its a single line)

Reply:
My idea was to try make sure it was explicit for future contributions that these two places have to have vectors that are exactly the same, but that is fair.

https://github.com/OpenLoco/OpenLoco/wiki/coding-style uses single line multiline comments twice, so that could need to be updated if that is the case.

Reply:
Ah yeah we probably should update it, we usually just do multiline comments as multiple lines of `//`. I don't think I've ever looked at that coding-style except when I first joined haha, I have just been trying to copy the current style in the code